tests/data/python/sample_1/helloworld.py

```diff
@@ -2,6 +2,10 @@
 # dummy comment second line
 def main():
     print("Hello World")
+    # this is added for semgrep test
+    test_var = "aaa"
+    if test_var is "bbb":
```

Comment:
```suggestion
    if test_var == "bbb":
```

`is` compares singletons which strings are not guaranteed to be singleton

python internally caches small strings so in most cases its true but its def not true for long strings:

```python
>>> "a" is "a"
<stdin>:1: SyntaxWarning: "is" with a literal. Did you mean "=="?
True

>>> "a" * 2**20 is "a" * 2**20
False
```

Reply:
we need this to be `is` for the semgrep rule to trigger correctly (`rule,yaml`)